README.md

```diff
@@ -223,32 +270,6 @@ bundle exec bugsnag-maze-runner features/something.feature
 Add the `--verbose` option to print script output and a trace of what Ruby file
 is being run.
 
-## Logging data on scenario failures
```

Comment:
Sections L226-251 deliberately removed?

Reply:
We no-longer handle failures this way, although I believe we do have some sort of failure mechanism so I'll have a look into that.

Reply:
I've added information about how logging works now in the `troubleshooting` section.